setup.py

```diff
@@ -28,4 +28,5 @@
           packages=['aexpect',
                     'aexpect.utils'],
           scripts=['scripts/aexpect_helper'],
-          test_suite='tests')
+          test_suite='tests',
+          install_requires=['six'])
```

Comment:
This is correct.  But given the current Python 3 only status, we must drop six ASAP.  I've opened https://github.com/avocado-framework/aexpect/issues/83.